molecule-gradle-plugin/src/main/java/app/cash/molecule/gradle/MoleculePlugin.kt

```diff
@@ -36,8 +36,8 @@ import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet.Companion.COMMON_MAIN_
 import org.jetbrains.kotlin.gradle.plugin.SubpluginArtifact
 import org.jetbrains.kotlin.gradle.plugin.SubpluginOption
 
-private const val extensionName = "molecule"
-private const val moleculeRuntime = "app.cash.molecule:molecule-runtime:$moleculeVersion"
+private const val EXTENSION_NAME = "molecule"
```

Comment:
Java called...

Reply:
Yeah I hate it, but https://kotlinlang.org/docs/coding-conventions.html#property-names